include/mysql_connection.h

```diff
@@ -4,6 +4,9 @@
 #include "proxysql.h"
 #include "cpp.h"
 
+#include "../deps/json/json.hpp"
+using json = nlohmann::json;
```

Comment:
Normally, I'm against namespace pollution in header files, but I understand that for this particular one, it shouldn't be any long term trouble.